doc/dev/config.rst

```diff
@@ -57,7 +57,7 @@ Reading configuration values
 There are two ways for Ceph code to get configuration values. One way is to
 read it directly from a variable named "g_conf," or equivalently,
 "g_ceph_ctx->_conf." The other is to register an observer that will be called
-every time the relevant configuration values changes.  This observer will be
+every time the relevant configuration values changes. This observer will be
```

Comment:
@shun-s @joscollin why is this considered a typo? is this an accepted guide line to have a single space to start a new sentence in our document? or is it recommended by some best practice ? if yes, for what reason?

for more counter examples, see https://github.com/ceph/ceph/pull/16611/files